suite-native/state/src/extraDependencies.ts

```diff
@@ -25,7 +25,7 @@ const transports = transportsPerDeviceType[deviceType];
 
 export const extraDependencies: ExtraDependencies = mergeDeepObject(extraDependenciesMock, {
     selectors: {
-        selectEnabledNetworks: () => supportedNetworkSymbols,
+        selectEnabledNetworks: () => supportedPortfolioTrackerNetworkSymbols,
```

Comment:
@vytick Why did you put portfolio tracker coins here? This selector shoudn't be related to portfolio tracker.